inst/rmarkdown/templates/NPS_DRR/skeleton/skeleton.Rmd

```diff
@@ -75,7 +75,7 @@ Note that if you need multiple paragraphs or line breaks you can generate them u
 The abstract should succinctly describe the study, the assay(s) performed, the resulting data, and their reuse potential, but should not make any claims regarding new scientific findings. No references are allowed in this section."
 
 # DataStore reference ID for the data package associated with this report. You must have at least one data package.Eventually, we will automate importing much of this information from metadata.
-dataPackageRefID <- c(0000000)
+dataPackageRefID <- c(9999999)
```

Comment:
Thanks! I hope this is an improvement but it's hard to anticipate multiple users mindframes.